src/SwipeProfiles.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './Register.css';
+import {browserHistory} from 'react-router';
+
+class SwipeProfiles extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            firstName = '',
+            description = '',
```

Comment:
might need to add age as well, would be good to have

Reply:
ya still working on it, so it will have name, interest, description, gender, workplace all shown.